airbyte-integrations/connectors/source-zoho-crm/source_zoho_crm/spec.json

```diff
@@ -0,0 +1,70 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/zoho-crm",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Zoho Crm Configuration",
+    "type": "object",
+    "required": [
+      "credentials", "environment", "dc_region", "edition"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "credentials": {
```

Comment:
if oauth is the only way for authenticating, I recommend that we don't place it in a nested object `credentials`, let's just put it at the top level

Reply:
done